### ISSUE TITLE
example of sheet music generation feature

### ISSUE THREAD

**Issue by francqz31** [rakuri255/UltraSinger#163]
I noticed the new sheet music generation feature . Does this convert midi to music score ? or a whole mp3/ raw vocals/song to music score.
is this like https://github.com/chrisdonahue/sheetsage ? or is it better/different?

**Comment by rakuri255** [comment]
Yes it converts midi or the ultrastar song into music score. But only the vocals+lyric. 
I dont know if it does betten then sheetsage. I would say UltraSinger do it worse 😄
